backend/index.ts

```diff
@@ -30,13 +31,30 @@ app.use(cors({ credentials: true, origin: "*" }));
 // routes middleware
 app.use("/api", router);
 
-// TODO: ADD SWAGGER
-// swagger
-// app.use("/api-docs", swaggerUI.serve, swaggerUI.setup(swDocument));
-
 // turn on the server
 const port = process.env.PORT || 3001;
 
 app.listen(port, () => {
   console.log(`Server is running on port ${port}`);
 });
+
+// swagger
+const options = {
+  definition: {
+    openapi: "3.0.0",
+    info: {
+      title: "Library API",
+      version: "1.0.0",
+      description: "MapUA API",
+    },
+    servers: [
+      {
+        url: "http://localhost:3001",
```

Comment:
Create a varaibale HOST inside .env file (and add this variable to .envExample file) and assign http://localhost:3001 to this variable
then use process.env.HOST